zgw/zaken-api/src/main/kotlin/nl/nlportal/zakenapi/client/ZakenApiClient.kt

```diff
@@ -67,10 +68,18 @@ class ZakenApiClient(
         return "${zakenApiConfig.url}/zaken/api/v1/zaken/$zaakId"
     }
 
+    fun getZaakTypeUrl(zaakTypeId: Any): String {
```

Comment:
Wat is het verschil tussen de functie hierboven?

Reply:
Is inderdaad incorrect, moet de zaakType url zijn. Ga het aanpassen